src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -252,14 +252,14 @@ private void getMatchingAttributes() {
                     try {
                         if (jmxAttribute.match(conf)) {
                             jmxAttribute.setMatchingConf(conf);
-                            metricsCount += jmxAttribute.getMetricsCount();
+                            metricMatchCount += 1;
```

Comment:
For `JMXComplexAttribute`s, the number of metric returned can be higher than `1`.